src/platform_impl/ios/app_state.rs

```diff
@@ -776,16 +772,13 @@ pub unsafe fn handle_main_events_cleared() {
 
     // User events are always sent out at the end of the "MainEventLoop"
     handle_user_events();
-    handle_nonuser_event(Event::EventsCleared);
+    handle_nonuser_event(Event::MainEventsCleared);
 
     let mut this = AppState::get_mut();
     let redraw_events = this
         .main_events_cleared_transition()
         .into_iter()
-        .map(|window| Event::WindowEvent {
-            window_id: RootWindowId(window.into()),
-            event: WindowEvent::RedrawRequested,
-        });
+        .map(|window| Event::RedrawRequested(RootWindowId(window.into())));
```

Comment:
I think we may need to send a `RedrawEventsCleared` after this.

Reply:
Good catch. All the other platforms send `RedrawEventsCleared` afterward.